src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp

```diff
@@ -35,6 +37,8 @@ class SubgraphBaseTest : public ov::test::TestsCommon {
     virtual void compile_model();
     virtual void init_ref_function(std::shared_ptr<ov::Model> &funcRef, const std::vector<ov::Shape>& targetInputStaticShapes);
     virtual void generate_inputs(const std::vector<ov::Shape>& targetInputStaticShapes);
+    virtual ov::test::utils::InputsMap get_input_map();
```

Comment:
I think that is extra code (we use these map to generate and compare data. Also they are defined out of the class)... Let's discuss offline.